lfs/gitscanner_log.go

```diff
@@ -341,6 +342,17 @@ func (s *logScanner) scan() (*WrappedPointer, bool) {
 }
 
 func (s *logScanner) setFilename(name string) {
+	// Trim last character if it's a quote
+	if len(name) > 0 && name[len(name)-1] == '"' {
+		name = name[:len(name)-1]
+	}
+
+	// Convert octals to proper UTF-8 code
+	unquotedName, err := strconv.Unquote(`"` + name + `"`)
+	if err == nil {
+		name = unquotedName
+	}
```

Comment:
I think we still have one problem here, which is that this should only apply to strings which were quoted in the first place.  If they're not quoted, then we don't want to unquote them, since that could result in incorrect results.

So this block should go within the previous `if` statement.

Reply:
Not sure about this. The if statement before is just to catch the ending `"` of the second matching group because I could not find a regex that excludes the last `"` as lookaheads are not supported in go.

But anyway, I think it's a no-op for non quoted strings, no? [See this example](https://go.dev/play/p/CZz4nYz1A3v)

Reply:
Yeah, I think you're right.  We wouldn't end up with a backslash escape unless we were quoted, and if we had a backslash at all, we'd have an escape and quoting.